build/common.sh

```diff
@@ -97,6 +97,7 @@ readonly GCS_STAGE="${LOCAL_OUTPUT_ROOT}/gcs-stage"
 # The set of master binaries that run in Docker (on Linux)
 # Entry format is "<name-of-binary>,<base-image>".
 # Binaries are placed in /usr/local/bin inside the image.
+# TODO: Make base images for multiple server platforms
```

Comment:
@zmerlynn Do you have some suggestions how to nicely handle this?
We need to be able to specify different base images for different arches